features/bootstrap/HttpHeaderContext.php

```diff
@@ -25,24 +24,12 @@ public function __construct(Request $request)
     }
 
     /**
-     * Sets the default Accept HTTP header to null (workaround to artificially remove it).
-     *
-     * @AfterStep
-     */
-    public function removeAcceptHeaderAfterRequest(AfterStepScope $event)
-    {
-        if (preg_match('/^I send a "[A-Z]+" request to ".+"/', $event->getStep()->getText())) {
-            $this->request->setHttpHeader('Accept', null);
-        }
-    }
-
-    /**
-     * Sets the default Accept HTTP header to null (workaround to artificially remove it).
+     * Sets the default Accept HTTP header to application/ld+json before each scenario.
      *
      * @BeforeScenario
      */
     public function removeAcceptHeaderBeforeScenario()
     {
-        $this->request->setHttpHeader('Accept', null);
+        $this->request->setHttpHeader('Accept', 'application/ld+json');
```

Comment:
as you removed almost every case in steps, I suggest we remove this method as well

Reply:
Shouldn't we remove the entire class then ? (and remove all references to it in behat.yml.dist)